packages/runtime/container-runtime/api-report/container-runtime.legacy.alpha.api.md

```diff
@@ -116,7 +116,6 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents
     // (undocumented)
     get IFluidHandleContext(): IFluidHandleContext;
     get isDirty(): boolean;
-    protected _loadIdCompressor: Promise<void> | undefined;
```

Comment:
Because 'createIdCompressor' is no longer async, we do not need a promise to track the "load in progress" state.